readthedocs/organizations/forms.py

```diff
@@ -125,41 +125,43 @@ class OrganizationSignupForm(SettingsOverrideObject):
     _default_class = OrganizationSignupFormBase
 
 
-class OrganizationOwnerForm(forms.ModelForm):
+class OrganizationOwnerForm(forms.Form):
 
     """Form to manage owners of the organization."""
 
-    class Meta:
-        model = OrganizationOwner
-        fields = ['owner']
-
-    owner = forms.CharField(label=_('Email address or username'))
+    user = forms.CharField(label=_("Email address or username"))
```

Comment:
I'd use `email_or_username` here to make it clear that this is the user's input while reading the code. Otherwise, it's easy to get confused by `user`, `username` and whether the `user` is a User object or the input from the person.

Here, and in the following form as well --which is where I got confused with the validator.